docs/OperatorBestPractices.md

```diff
@@ -52,8 +85,8 @@ apiVersion: apiextensions.k8s.io/v1beta1
 metadata:
   name: databases.postgresql.baiju.dev
   annotations:
-    servicebindingoperator.redhat.io/status.dbConfigMap.password: 'binding:env:object:secret'
-    servicebindingoperator.redhat.io/status.dbConfigMap.username: 'binding:env:object:configmap'
+    servicebindingoperator.redhat.io/status.dbConfigMap-password: 'binding:env:object:secret'
```

Comment:
perhaps we could share more details about what is the semantic of these lines?

Reply:
Good idea! 
Fixed up an annotation too. 
https://github.com/redhat-developer/service-binding-operator/pull/618/commits/2cf29e9a568009a4bf20dd40afb3dbf256f02b42